packages/math/src/integers.ts

```diff
@@ -55,7 +55,7 @@ export class Uint32 implements Integer, WithByteConverters {
 
   public static fromString(str: string): Uint32 {
     if (!str.match(/^[0-9]+$/)) {
-      throw new Error("Invalid string format");
+      throw new Error("Invalid string format. Found: " + str);
```

Comment:
Can you put those in quotes? Probably easier to write with a template string.

```ts
throw new Error(`Invalid string format. Found: '${str}'`);
```

Thanks

Reply:
I had also thought about it, but I based it on https://github.com/cosmos/cosmjs/blob/2c3b27eeb3622a6108086267ba6faf3984251be3/packages/math/src/integers.ts#L75. I will adapt to reflect your suggestion.